consensus/src/block_graph.rs

```diff
@@ -110,6 +110,21 @@ impl From<ActiveBlock> for ExportActiveBlock {
     }
 }
 
+impl From<Box<ActiveBlock>> for ExportActiveBlock {
```

Comment:
Why did you boxed active block ?

Reply:
For almost the same error as #1265.

In the enum bellow, the variant Active<ActiveBlock> is a lot larger in memory than the other variants.
So in memory, because the data has to be aligned, all other variants takes the same size than Active<ActiveBlock> and keep useless space (an offset) to be aligned.

```rust
enum BlockStatus {
    Incoming(HeaderOrBlock),
    WaitingForSlot(HeaderOrBlock),
    WaitingForDependencies {
        header_or_block: HeaderOrBlock,
        unsatisfied_dependencies: BlockHashSet, // includes self if it's only a header
        sequence_number: u64,
    },
    Active(ActiveBlock), // I replaced it by Active<Box<ActiveBlock>>
    Discarded {
        header: BlockHeader,
        reason: DiscardReason,
        sequence_number: u64,
    },
}
```

Note: There is also some macros used to tell specifically "don't align that!" but it will produce a slower execution.